org.pitivi.Pitivi.json

```diff
@@ -0,0 +1,497 @@
+{
+    "app-id": "org.pitivi.Pitivi",
+    "branch": "1.0",
```

Comment:
Set it to `stable` or remove altogether, our CI sets it on its own.

Reply:
Done.